scale/job/seed/manifest.py

```diff
@@ -417,7 +417,10 @@ def needs_input_metadata(self):
         :rtype: bool
         """
 
-        return 'INPUT_METADATA_MANIFEST' in self.get_input_files()
```

Comment:
We can entirely remove the needs_input_metadata check against the manifest. May still want evaluation for if no input at all exists. As long is it can create empty metadata json, we are good.

Reply:
done